linera-execution/tests/wasm.rs

```diff
@@ -24,10 +24,10 @@ use test_case::test_case;
 /// called correctly and consume the expected amount of fuel.
 ///
 /// To update the bytecode files, run `linera-execution/update_wasm_fixtures.sh`.
-#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::Wasmer, 48_382, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
-#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::WasmerWithSanitizer, 49_042, ExecutionRuntimeConfig::Synchronous; "wasmer_with_sanitizer"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 49_042, ExecutionRuntimeConfig::Synchronous; "wasmtime"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 49_042, ExecutionRuntimeConfig::Synchronous; "wasmtime_with_sanitizer"))]
+#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::Wasmer, 61_530, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
+#[cfg_attr(feature = "wasmer", test_case(WasmRuntime::WasmerWithSanitizer, 62_038, ExecutionRuntimeConfig::Synchronous; "wasmer_with_sanitizer"))]
+#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 62_038, ExecutionRuntimeConfig::Synchronous; "wasmtime"))]
+#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 62_038, ExecutionRuntimeConfig::Synchronous; "wasmtime_with_sanitizer"))]
```

Comment:
(That's much more than I'd expect…?)

Reply:
Yeah, it was certainly a surprise to me. I need to investigate more why this is happening.